[Enhancement] - ServiceNow - Non-Interactive OAuth2 for v2 Integration (#42975)
demisto/content: Packs/ServiceNow/Integrations/ServiceNow_CMDB/ServiceNow_CMDB_description.md

#### OAuth 2.0 Authorization
To use OAuth 2.0 authorization perform the following steps:
1. Login to your ServiceNow instance and create an endpoint for XSOAR to access your instance. (See [Snow OAuth](https://docs.servicenow.com/bundle/xanadu-platform-security/page/administer/security/concept/c_OAuthApplications.html) for more information).

> 2. Copy the **Client Id** and **Client Secret** (press the lock next to the client secret to reveal it) that were automatically generated when creating the endpoint into the **Client ID** and **Client Secret** fields of the instance configuration.

Reviewer: ```suggestion
2. Copy the **Client Id** and **Client Secret** (click the lock next to the client secret to reveal it) that were automatically generated when creating the endpoint into the **Client ID** and **Client Secret** fields for the integration instance configuration.
```

> 3. (Recommended) Enter the ServiceNow account's **Username** and **Password** in the instance configuration. When provided, the integration will automatically perform the initial OAuth login on first use — no manual ***!servicenow-cmdb-oauth-login*** command is needed. It will also automatically renew the refresh token when it expires.

Reviewer: ```suggestion
3. (Recommended) Enter the ServiceNow account **Username** and **Password** in the integration instance configuration. The integration will then automatically perform the initial OAuth login on first use and will automatically renew the refresh token when it expires.
```

4. Select the **Use OAuth Login** checkbox and click **Done**.

> 5. If you did **not** provide **Username** and **Password** in step 3, run the command ***!servicenow-cmdb-oauth-login*** from the XSOAR CLI and fill in the username and password of the ServiceNow instance. This step generates and saves in the integration context a refresh token to the ServiceNow instance and is required only the first time you configure a new instance in the XSOAR platform. If you provided **Username** and **Password**, this step is handled automatically.

Reviewer: ```suggestion
5. If you did **not** provide **Username** and **Password** in step 3, run the command ***!servicenow-cmdb-oauth-login*** from the Cortex XSOAR CLI and fill in the username and password of the ServiceNow integration instance. This step generates and saves in the integration context a refresh token to the ServiceNow instance and is required only the first time you configure a new instance in the Cortex XSOAR platform. If you provided **Username** and **Password**, this step is handled automatically.
```

Relevant for XSIAM?

> 6. (Optional) Test the created instance by running the ***!servicenow-cmdb-oauth-test*** command.

Reviewer: ```suggestion
6. (Optional) Test the created integration instance by running the ***!servicenow-cmdb-oauth-test*** command.
```
